metal/eip_controlplane_reconciliation.go

```diff
@@ -224,7 +226,8 @@ func newControlPlaneEndpointManager(eipTag, projectID string, deviceIPSrv packng
 // the `default/kubernetes` service
 func (m *controlPlaneEndpointManager) reconcileServices(ctx context.Context, svcs []*v1.Service, mode UpdateMode) error {
 	if m.eipTag == "" {
-		return errors.New("elastic ip tag is empty. Nothing to do")
```

Comment:
I'm hitting this error in a cluster I just brought up under k8s 1.23, haven't looked into why yet.
Can you explain the change in behavior here?

Reply:
(why is this not an error)

Reply:
Yeah. The way it is defined, if you set an EIP tag, then you want CPEM to manage it; if you do not set an EIP tag, then you are telling CPEM, "there is nothing for you to manage at the control plane EIP level." Lack of EIP tag = "nothing to do here". That definitely is _not_ an error, and never should have been treated as such.

Reply:
Don't ask how many people that led astray. 🤦‍♂️

Reply:
I sort of did `"Nothing to do."` as an indication that this is not an error, just nothing that needs to be done here. The correct answer to that is, "so don't make it an error!" 😬